File: Concurs Inot Java/ProbaParticipant.java
package me.model;

import java.io.Serializable;

public class ProbaParticipant extends Entity<String> implements Serializable {
    private Integer idProba;
    private Integer idParticipant;
    public ProbaParticipant(Integer idProba, Integer idParticipant){
        super.setId(idProba.toString() + "." + idParticipant.toString());
        this.idProba = idProba;
        this.idParticipant = idParticipant;
    }

    public Integer getIdProba() { return idProba; }

    public Integer getIdParticipant() { return idParticipant; }
}

File: Concurs Inot C#/ProbaParticipant.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Problema_3_CSharp.me.model
{
    [Serializable]
    public class ProbaParticipant : Entity<string>
    {
        public int IdProba { get; }
        public int IdParticipant { get; }
        public ProbaParticipant(int idProba, int idParticipant)
        {
            base.Id = idProba.ToString() + "." + idParticipant.ToString();
            IdProba = idProba;
            IdParticipant = idParticipant;
        }
    }
}
